scss/_custom-forms.scss

```diff
@@ -210,6 +210,8 @@
 .custom-file-input {
   min-width: $custom-file-width;
   max-width: 100%;
+  padding-top: $custom-file-padding-y;
+  padding-bottom: $custom-file-padding-y;
```

Comment:
As I said before, it's to make the invisible input the same height as the visible .custom-file-control. Without it, dragging something onto the lower third of the button doesn't work.